src/client/utils/use-form.ts

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+
+export const useForm = <T = any>(initialValues: any) => {
+	const [values, setValues] = useState<T>(initialValues);
+
+	const handleChanges = (
```

Comment:
```suggestion
	const handleChanges: () => void = (
```
If this doesn't break your code, theoretically this should have typing!

Reply:
File is going to be deleted